packages/vscode-plugin/.vscodeignore

```diff
@@ -5,3 +5,12 @@
 !CHANGELOG.md
 !bin
 !icon.png
+
+# Ignore build files
+build/
+
+# Ignore test files
+tests/
+
+# Include media directory
+!media/
```

Comment:
I think this line is the only thing that's needed to actually fix the issue.

As an explanation, the first line of this file is `**/**` which tells `@vscode/vsce` to ignore everything when packaging and all the subsequent lines starting with `!` tell it what is an exception to that. So, when packaging the extension without this line this is the output:

![pr-1077-1](https://github.com/user-attachments/assets/e3757a1a-a3b9-413e-81d1-c2ab51f3ba33)

But with this line included, this is the output:


![pr-1077-2](https://github.com/user-attachments/assets/2934ffc9-cd7e-4361-8e88-df91481ef70a)

Reply:
You can check the latest VS Code extension release action and see that this is indeed the case: https://github.com/Automattic/harper/actions/runs/14476652533/job/40603810278#step:7:87